### PR TITLE
Add route for extension links

### PR DESCRIPTION
**Pull request by dylanjw**
See status-im/status-react#6199

Adds a route for extension links in the browser.  Tested manually in status dev mode.

### CODE REVIEW

File: tests/run.sh [hunk 1]
```diff
@@ -48,3 +48,6 @@ shakedown GET /browse/www.test.com/blah/blah
 shakedown GET /health
   status 200
   contains 'OK'
+
+shakedown GET /extension/blah
```
Comment (jeluard) [inline tests/run.sh]: Why blah?

Comment (dylanjw) [reply tests/run.sh]: No particular reason. I copied the test for `user` links which also has "blah".  They could be replaced with realistic values, but then there is the fact that those values are not validated, and putting realistic values might confuse a reader into thinking that they are.

Comment (jeluard) [reply tests/run.sh]: Sounds great!

**Comment by jeluard** [comment]
@jakubgs What do we have to do if we want this change to be deployed?

**Comment by cammellos** [comment]
@jeluard if things are as they were before, this is automatically deployed on successful build

**Pull request merged**